### ISSUE TITLE
Title is harcoded in the HTML

### ISSUE THREAD

**Issue by Natrium729** [inkle/inky#168]
With the latest release of Inky, it is possible to add a byline with `# author: Natrium729`. The byline is added to the HTML page by the JavaScript so it is possible to update it by exporting story.js only.

However, the title set with `# title: Hello World` is hardcoded into the HTML while exporting for the web. Because of this, it can't be updated by only exporting story.js.

Quite a minor issue, but can be a bit tiring if you set the title long after having made modifications to the CSS/JS.

**Comment by joningold** [comment]
But you only have to edit the html file once, right? Or am I
misunderstanding?

**Comment by joethephish** [comment]
Yeah, it’s just that the new author approach is very slightly better!

**Comment by paulloz** [comment]
`meta` and `title` tags should anyway always be hard defined in the html file, as they're parsed by robots (often) without javascript support.

**Comment by lunarcloud** [comment]
That said, a 'title' tag would be helpful so that the name of the export folder doesn't have to match the actual title - since then our titles are initially limited by what file systems support for folder names.

**Comment by lunarcloud** [comment]
I think that @joethephish added this aaa75425b146ec962900ffd9cf6a37e307c83f6c (See [line 460 of inkProject.js](https://github.com/inkle/inky/blob/c0de10d2c1f4edc979bf1a9b6945f088c69542a3/app/renderer/inkProject.js#L460) ) but I couldn't get it to work.
